Readme.md

```diff
@@ -1,17 +1,62 @@
-[![Build Status](https://travis-ci.org/MakingSense/WebApiCore-Seed.svg?branch=master)](https://travis-ci.org/MakingSense/WebApiCore-Seed)
-# WebapiCore-seed
-
-## Technologies
-* Server Side
-    * Asp.Net Web Api Core
-    * Entity Framework(as DAL)
-    * Net Core DI
-    * Xunit
+# WebapiCore-seed [![Build Status](https://travis-ci.org/MakingSense/WebApiCore-Seed.svg?branch=master)](https://travis-ci.org/MakingSense/WebApiCore-Seed)
+
+## Prerequisites
+
+.NET Core SDK
+
+A text processor / gui
```

Comment:
Sorry, I did not express myself well. I was not referring to the markup bullet point but rather to the text processor/GUI/IDE requirement. I think it's pretty obvious you will need one to crack some code. But I do like having the prerequisites title with the SDK requirement.